cprover_bindings/src/goto_program/typ.rs

```diff
@@ -343,6 +347,7 @@ impl Type {
             IncompleteStruct { .. } => unreachable!("IncompleteStruct doesn't have a sizeof"),
             IncompleteUnion { .. } => unreachable!("IncompleteUnion doesn't have a sizeof"),
             InfiniteArray { .. } => unreachable!("InfiniteArray doesn't have a sizeof"),
+            Integer => st.machine_model().int_width,
```

Comment:
This doesn’t have a sizeof, should be handled like the line before.

Reply:
Done.